content/docs/installation/_index.md

```diff
@@ -7,5 +7,5 @@ weight: 1
 The orchestrator solution is made of various independent components where each has its own
 installation method. At the moment we support a specialized helm chart to install everything
 from scratch on OpenShift or on Kubernetes. The installation is modular so if any of the pieces
-is already installed it can be made optional (more in the helm chart README.md below.
+is already installed it can be made optional (more in the helm chart README.md below).
```

Comment:
Not sure if mentioning `README.md` can confuse the user. From the website, the user hasn't the perception of the README files.
Not a strong opinion on this, but I'd like to highlight this :)

Reply:
tried to address this. WDYT?

Reply:
Great! thanks